### ISSUE TITLE
Add form that submits issues to Runelite github

### ISSUE THREAD

**Issue by arthurtyukayev** [runelite/runelite.net#150]
This could open up suggestions to more people that aren't really familiar with Github and/or Github Issues.

They'd be anonymous, so some sort of flood protection would be recommended (Captcha), etc.

**Comment by Nightfirecat** [comment]
There's not really any way to submit issues without an account.

**Comment by Alexsuperfly** [comment]
could have it post through one that you setup to handle these

**Comment by Abextm** [comment]
1) It wouldn't provide a way to look for duplicates and we have better things to do than close duplicates
2) We would have no way to get in contact with the author

**Comment by arthurtyukayev** [comment]
That's true, it would lead to more issues created. That being said, it could be made to search the issues for some duplicates, and display those. In addition to that, upon issue creation it could display the link to the created issue so they could monitor the issue response?

**Comment by Abextm** [comment]
Or we could just make them post to GitHub which doesn't require us to write and maintain that.

**Comment by trimbe** [comment]
I think we have enough people that are willing to use Github to submit issues that this isn't really necessary.

**Comment by Monsterxsync** [comment]
The github issues seem to be good enough, theres already a link in the client to report issues/make suggestions.
